.golangci.yml

```diff
@@ -3,17 +3,21 @@ run:
   timeout: 10m
   sort-results: true
   allow-parallel-runners: true
-  exclude-dir: testutil/testdata
+  skip-dirs:
+    - crypto
+    - testutil/testdata
 
 linters:
   disable-all: true
   enable:
+    - bodyclose
```

Comment:
As @tac0turle said let's not introduce new linters until everything is cleaned-up.

Reply:
o boy then I made a couple of ill considered moves.  moment please sir